src/data/actions/courseOptions.js

```diff
@@ -38,15 +40,15 @@ function fetchCourseOptions(id) {
 
         // Confirm it looks vaguely correct
         if (!course || !('actions' in course)) {
-          throw Error('Did not understand response.');
```

Comment:
Why delete this poor period?

Reply:
Adding it in later because our API error messages don't have periods, and want it to be consistent without saying `Some full error string: Did not understand response..`